tests/virt/node/migration_and_maintenance/test_vm_unscheduled_node.py

```diff
@@ -54,26 +62,24 @@ def unscheduled_node_vm(
     indirect=True,
 )
 @pytest.mark.polarion("CNV-4157")
-def test_schedule_vm_on_cordoned_node(nodes, data_volume_scope_function, unscheduled_node_vm):
+def test_schedule_vm_on_cordoned_node(nodes, data_volume_scope_function, unscheduled_node_vm, worker_node1):
```

Comment:
`nodes` not used

also, nit: pls reorder - `worker_node1` should be first since unscheduled_node_vm depends on it